logstash-output-http.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-output-http'
-  s.version         = '3.1.1'
+  s.version         = '4.0.0'
```

Comment:
@suyograo the main reason I did 4.0.0 is that the new retry behavior is on by default. I want people to consider that since if they expect failures they may find that the output stalls.